lib/Importer/Caffe2.cpp

```diff
@@ -270,9 +270,20 @@ void caffe2ModelLoader::loadOperator(const caffe2::OperatorDef &op) {
 
     Node *node = G_.createConcat(opName, inputs, channel);
 
+    unsigned_t addAxis = dict.count("add_axis") ? loadInt(dict["add_axis"]) : 0;
+
+    Node *concatOutput = node;
```

Comment:
nit: you can remove this line. Either rename above `Node *node=` to `Node *concatOutput=`, or use `node` instead of `concatOutput` below.

Reply:
Done